docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -69,9 +69,17 @@ for a full reference of flags for this command:
 $ tctl sso configure github \
 --id=<Var name="GITHUB-CLIENT-ID"/> \
 --secret=$(cat client-secret.txt) \
---teams-to-roles=<Var name="ORG-NAME,GITHUB-TEAM,access,editor"/> \
+--teams-to-roles=<Var name="ORG-NAME,github-team,access,editor"/> \
 > github.yaml
 ```
+<Admonition type="tip">
+  GitHub organizations and teams should be referred to by their slug, not display name.
+  To create the slug, GitHub replaces special characters in the name string, changes all words to lowercase,
+  and replaces spaces with a `-` separator. For example, `My TEam Näme` would become `my-team-name`.
+  You can confirm the slug in GitHub Web application URLs or via the GitHub API.
+  Example: `example-team` in the URL `https://github.com/orgs/Org-Example/teams/example-team`
+  matches to the slug for team `Example Team`.
```

Comment:
I like the wording that you used on the troubleshooting page better:

> Example: navigate to the team `My Team` in the GitHub web application.
 The URL `https://github.com/orgs/org-name/teams/my-team`
 shows the slug is `my-team`. Update the teams to roles 

I would just repeat that here.